### PR TITLE
fix: add Anish Lohiya to the list

### PR DESCRIPTION
**Pull request by AnishLohiya**
## Changes proposed

Added AnishLohiya Profile 

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/859"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Please revert the changes in the `package-lock.json` and `package.json` files. Let us know if you need any help! :)

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   "name": "linkfree",
-  "version": "0.54.1",
+  "version": "0.54.4",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.54.1",
+      "version": "0.54.4",
```
Comment (krishguptadev) [inline package-lock.json:9]:
```suggestion
      "version": "0.54.1",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "linkfree",
-  "version": "0.54.1",
+  "version": "0.54.4",
```
Comment (krishguptadev) [inline package-lock.json:3]:
```suggestion
  "version": "0.54.1",
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you remove the endline in package.json

File: public/data/AnishLohiya.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "url": "https://github.com/AnishLohiya",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AnishLohiya169",
+      "icon": "twitter"
+    }
+  ]
+}
```
Comment (syedsohan) [inline public/data/AnishLohiya.json:18]:
```suggestion
}
```

**Review by syedsohan — CHANGES_REQUESTED** [review]
Can you add an endline after line number 18!

**Review by syedsohan — APPROVED** [review]
Looks good 👍

File: public/data/AnishLohiya.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (krishguptadev) [inline public/data/AnishLohiya.json:19]:
```suggestion
```

Comment (AnishLohiya) [reply public/data/AnishLohiya.json:19]: @syedsohan  Said to add an endline

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**